assets/js/src/public.jsx

```diff
@@ -85,6 +85,10 @@ jQuery(($) => {
     const form = formDiv.find('form');
     const position = form.data('position');
     formDiv.addClass(`mailpoet_form_position_${position}`);
+    const animation = form.data('animation');
+    if (animation !== '') {
+      formDiv.addClass(`mailpoet_form_animation_${animation}`);
+    }
     const background = form.data('background-color');
     formDiv.css('background-color', background || 'white');
```

Comment:
Not related to this PR but I think we can remove these lines since we already set this in CSS we generate here: https://github.com/mailpoet/mailpoet/blob/master/lib/Form/Util/Styles.php#L65-L78
We can probably also remove the data attribute `data-background-color="<%= backgroundColor %>"`